yamoveset/panels/main.py

```diff
@@ -92,6 +98,7 @@ def build_tree(self):
                 root, f'{entry.index}: {KNOWN_ENTRIES.get(entry.index, "Unknown")}', data=entry)
         self.save.Enable()
         self.paste.Enable()
+        self.add.Enable()

```

Comment:
Should be disabled if there is no copied data.

Reply:
clicking on Paste will also not show any message, so i copied the same behaivor
but i guess its a bug

Reply:
Pretty much.  As usual for coding, I hate myself from 2 years ago.